.github/workflows/deploy.yml

```diff
@@ -0,0 +1,141 @@
+          password: ${{ secrets.CLOUD_PASSWORD }}
+          dockerfile: dev/docker/Dockerfile
+          buildoptions: "--compress"
+
+  deploy-demo:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: chrnorm/deployment-action@releases/v1
+        if: github.ref == 'refs/heads/master'
+        name: Create GitHub deployment for demo
+        id: deployment
+        with:
+          token: "${{ github.token }}"
+          target_url: https://demo.europe-west1.gcp.storefrontcloud.io/api/
+          environment: production
+          initial_status: in_progress
+      # Deploy on demo
+      - name: Deploy on demo.europe-west1.gcp.storefrontcloud.io
+        if: github.ref == 'refs/heads/master'
+        run: |
+          if curl -s -H 'X-User-Id: ${{ secrets.DEPLOYMENT_USER }}' -H 'X-Api-Key: ${{ secrets.DEPLOYMENT_PASSWORD }}' -H 'Content-Type: application/json' -X POST -d '
+          {
+              "code":"demo",
+              "region":"europe-west1.gcp",
+              "apiContainerVersion":"${{ github.sha }}"
+          }' https://farmer.storefrontcloud.io/instances | grep -q '{"code":200,"result":"Instance updated!"}'; then
+            echo "Instance updated"
+          else
+            echo "Something went wrong during the update process..."
+            exit 1
+          fi
+
+      # Return status
+      - name: Update deployment status (success)
+        if: success()
+        uses: chrnorm/deployment-status@releases/v1
+        with:
+          token: "${{ github.token }}"
+          target_url: https://demo.europe-west1.gcp.storefrontcloud.io
+          state: "success"
+          description: Congratulations! The deploy is done.
+          deployment_id: ${{ steps.deployment.outputs.deployment_id }}
+
+      - name: Update deployment status (failure)
+        if: failure()
+        uses: chrnorm/deployment-status@releases/v1
+        with:
+          token: "${{ github.token }}"
+          target_url: https://demo.europe-west1-southeast1.gcp.storefrontcloud.io
+          description: Unfortunately, the instance hasn't been updated.
+          state: "failure"
+          deployment_id: ${{ steps.deployment.outputs.deployment_id }}
+
+  deploy-test:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: chrnorm/deployment-action@releases/v1
+        if: github.ref == 'refs/heads/develop'
+        name: Create GitHub deployment for test
+        id: deployment
+        with:
+          token: "${{ github.token }}"
+          target_url: https://test.europe-west1.gcp.storefrontcloud.io/api/
+          environment: demo
+          initial_status: in_progress
+      # Deploy on demo
+      - name: Deploy on test.europe-west1.gcp.storefrontcloud.io
+        if: github.ref == 'refs/heads/develop'
+        run: |
+          if curl -s -H 'X-User-Id: ${{ secrets.DEPLOYMENT_USER }}' -H 'X-Api-Key: ${{ secrets.DEPLOYMENT_PASSWORD }}' -H 'Content-Type: application/json' -X POST -d '
```

Comment:
Now, you should use CLOUD_USERNAME and CLOUD_PASSWORD instead of DEPLOYMENT_USER and DEPLOYMENT_PASSWORD